content/en/docs/concepts/architecture/nodes.md

```diff
@@ -49,24 +49,24 @@ try to create a Node from the following JSON manifest:
 }
 ```
 
-Kubernetes creates a Node object internally (the representation). Kubernetes checks
+Kubernetes creates a `Node` object internally (the representation). Kubernetes checks
 that a kubelet has registered to the API server that matches the `metadata.name`
-field of the Node. If the node is healthy (if all necessary services are running),
-it is eligible to run a Pod. Otherwise, that node is ignored for any cluster activity
+field of the `Node`. If the node is healthy (if all necessary services are running),
+it is eligible to run a pod. Otherwise, that node is ignored for any cluster activity
```

Comment:
nit: Question about using lowercase for Pod.

Reply:
Either `Pod` or pod is fine. I would pick `Pod`.